libaleth-interpreter/VM.cpp

```diff
@@ -1160,7 +1160,7 @@ void VM::interpretCases()
 
             updateIOGas();
 
-            m_SPP[0] = fromEvmC(getTxContext().chain_id);
+            m_SPP[0] = getTxContext().chain_id;
```

Comment:
How come we don't have to do the big endian conversion anymore?

Reply:
`chain_id` changed in evmc, now it's `uint64_t` (in native system endianness) and it can be converted implicitly to `u256`